src/main/java/core/basesyntax/enums/Operation.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.enums;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String code;
+
+    Operation(String code) {
+        this.code = code;
+    }
+
+    public String getCode() {
+        return code;
+    }
+
+    public static Operation fromCode(String code) {
+        for (Operation operation : values()) {
+            if (operation.getCode().equals(code)) {
+                return operation;
+            }
+        }
+        throw new IllegalArgumentException("No enum constant for code: " + code);
+    }
+}
```

Comment:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/63095184/1dfb5037-ba22-48bf-9337-15fafe3cb8ac)

Reply:
U can make like this

Reply:
done